installer.sh

```diff
@@ -4,17 +4,11 @@ original_wd=$(pwd)
 mytmpdir=$(mktemp -d 2>/dev/null || mktemp -d -t 'mytmpdir')
 git clone --depth=1 git@github.com:contentacms/contenta_jsonapi.git $mytmpdir
 cd $mytmpdir
-read -n 1 -p "Do you wish to install Contenta CMS (y/n)? " answer
```

Comment:
Let's not remove this, please. It's good practice to have people re-confirm when they are running an unknown script from the Internet.